repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEnumDefStoreV2.java

```diff
@@ -54,6 +54,10 @@ public AtlasVertex preCreate(AtlasEnumDef enumDef) throws AtlasBaseException {
 
         validateType(enumDef);
 
+        if(AtlasAbstractDefStoreV2.KEYWORDS_INVALID_FOR_TYPE_CREATION.contains(enumDef.getName())){
+            throw new AtlasBaseException(AtlasErrorCode.UNKNOWN_TYPENAME, enumDef.getName());
```

Comment:
I don't think unknown type name would fit the purpose. You can send error like Forbidden type: Can't pass builtin types. That would be more clear.